src/js/api/auth/login.test.js

```diff
@@ -0,0 +1,28 @@
+import { login } from "./login.js";
+import localStorageMock from "../../../../mocks/localStorage.mock.js";
+import userMock from "../../../../mocks/user.mock.js";
+
+const USER_DATA = userMock;
+
+const mockFetchSuccess = jest.fn().mockResolvedValue({
+  ok: true,
+  json: jest.fn().mockResolvedValue(USER_DATA),
+});
+
+global.localStorage = localStorageMock;
+
+global.fetch = mockFetchSuccess;
```

Comment:
```suggestion
beforeAll(() => {
    global.localStorage = localStorageMock;
    global.fetch = mockFetchSuccess;
  });
```

Consider setting the global mocks in an beforeAll hook: https://jestjs.io/docs/setup-teardown